scripts/globals/bcnm.lua

```diff
@@ -1131,7 +1131,9 @@ local function checkReqs(player, npc, bfid, registrant)
         end,
 
         [1154] = function() -- Quest: The Beast Within (BLU LB5)
-            return mainJob == xi.job.BLU and mainLevel >= 66
+            return mainJob == xi.job.BLU and mainLevel >= 66 and
+            xi.quest.getVar(player, xi.quest.log_id.AHT_URHGAN, xi.quest.id.ahtUrhgan.THE_BEAST_WITHIN, 'Prog') == 3 or
```

Comment:
You'll also need a check for registerReqs. Else anyone can start this battlefield whenever they want lol